src/components/AddItems.js

```diff
@@ -36,6 +42,25 @@ const AddItemsToList = () => {
       .get()
       .then((data) => {
         if (data.docs.length) {
+          const normalizeString = (str) => {
+            const nonWordCharactersAndUnderscores = /[\W_]/g;
+            return str
+              .toLowerCase()
+              .replace(nonWordCharactersAndUnderscores, '');
+          };
```

Comment:
This is just a small example, but I am *loving* how readable this code is! Seriously, a joy to read.